app.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask, jsonify
+import json
+import pdb
+
+# instantiate the app
+app = Flask(__name__)
+app.config.from_object(__name__)
+
+@app.route('/emoji', methods=['GET'])
+def get_emoji_list():
+    try:
+        with open('results.json') as json_file:
```

Comment:
FYI @nifengle, if this gets merged in before yours, we'll have two seed files in the app.

Reply:
I'm adding the seed file to gitignore

Reply:
don't add it to gitignore -- it's useful for devs to have it available for now. i'll point to wherever you put it @nifengle

Reply:
I don't want to commit the seed file, though